charts/spinnaker/templates/configmap/halyard-config.yaml

```
@@ -109,49 +109,29 @@ data:
     {{- end }}
 
     $HAL_COMMAND config provider kubernetes enable
-    {{- if .Values.kubeConfig.accountPerContext }}
-    {{- range $index, $context := .Values.kubeConfig.contexts }}
+    {{- range $accountName, $values := .Values.kubeConfig.accounts }}
```

```suggestion
    {{- range $accountName, $account := .Values.kubeConfig.accounts }}
```
Let's use a more descriptive name.

---

(Doubting whether we should also rename `accountName` -> `name`.)